registrations/tasks.py

```diff
@@ -265,6 +265,34 @@ def validate(self, registration):
             self.l.info("Registration object updated.")
             return False
 
+    def create_popi_subscriptionrequest(self, registration):
+        """
+        Creates a new subscription request for the POPI message set. This
+        message set tells the user how to access the POPI required services.
+        This sould only be sent for Clinic or CHW registrations.
+        """
+        if ('prebirth' not in registration.reg_type or
```

Comment:
We want to limit it to registrations from the clinic and the CHW lines. Currently, those two USSD lines only do `momconnect_prebirth` and `whatsapp_prebirth` registration types, so combined with limiting the authority type, this should limit it to only those two USSD lines.